_data/current.yml

```diff
@@ -266,3 +266,21 @@
   dates: "November 19-22, 2024"
   url: https://agiletestingdays.com/?utm_source=testingconferences
   status: Super Early Bird Pricing is Open until April 26, 2024
+  
+- name: The Premier Online Conference For Software Testing
+  location: online
+  dates: "May 21-22, 2024"
```

Comment:
Needs to be with the May Conferences

Reply:
Fixed by the last commit - > [Conferences sorted by date]